src/webui/www/private/views/preferences.html

```diff
@@ -2099,6 +2108,7 @@
 
             // libtorrent section
             settings.set('async_io_threads', $('asyncIOThreads').getProperty('value'));
+            settings.set('stop_tracker_timeout', $('stop_tracker_timeout').getProperty('value'));
```

Comment:
After changing above to camelCase, `$('stop_tracker_timeout')` should also be changed to `$('stopTrackerTimeout')`